pkg/routeagent_driver/handlers/calico/ippool_handler.go

```diff
@@ -66,11 +70,23 @@ var NewClient = func(restConfig *rest.Config) (calicocs.Interface, error) {
 
 var logger = log.Logger{Logger: logf.Log.WithName("CalicoIPPool")}
 
-func NewCalicoIPPoolHandler(restConfig *rest.Config, namespace string, k8sClient clientset.Interface) event.Handler {
+var gvrInstallations = schema.GroupVersionResource{
```

Comment:
We can make this public and share with unit tests:

```suggestion
var InstallationsGVR = schema.GroupVersionResource{
```

Reply:
Also similarly for the default `Installation` name:

`DefaultInstallationName = "default"`

Reply:
`gvrInstallations` renamed/changed to public.
Added `DefaultInstallationName`
Using them everywhere.